server/fleet/datastore.go

```diff
@@ -197,10 +197,12 @@ type Datastore interface {
 	ListHosts(ctx context.Context, filter TeamFilter, opt HostListOptions) ([]*Host, error)
 
 	// ListHostsLiteByUUIDs returns the "lite" version of hosts corresponding to
-	// the provided uuids and filtered according to the provided team filters.
-	// The "lite" version is a subset of the fields related to the host. See
-	// documentation of Datastore.HostLite for more information, or the
-	// implementation for the exact list.
+	// the provided uuids and filtered according to the provided team filters. It
+	// does include the MDMInfo information (unlike HostLite and
+	// ListHostsLiteByIDs) because listing hosts by UUIDs is commonly used to
+	// support MDM-related operations, where the UUID is often the only available
+	// identifier. The "lite" version is a subset of the fields related to the
```

Comment:
I did that out of necessity as what I think is pragmatic, but it is inconsistent with the other Lite endpoints. Happy to discuss other options if y'all feel uneasy about this. The only places this is called are both MDM related (`GetMDMAppleCommandResults` and those Run MDM command endpoints).

Reply:
sounds good to me